db/new_seeds/base/add_npq_registrations.rb

```diff
@@ -1,56 +1,114 @@
-  NewSeeds::Scenarios::NPQ
-    .new(lead_provider: npq_lead_providers.sample)
-    .build
-end
+  # Create accepted NPQ applications with participant profiles
+  # and a declaration
+  seed_quantity(:npq_application_with_declarations).times do
+    NewSeeds::Scenarios::NPQ
+      .new(
+        lead_provider: npq_lead_providers.sample,
+        cohort:,
+      )
+      .build
+      .accept_application
+      .add_declaration
+      .add_statement_line_items
+  end
 
-# Create accepted NPQ applications with participant profiles
-# and a declaration
-seed_quantity(:npq_application_with_declarations).times do
-  NewSeeds::Scenarios::NPQ
-    .new(lead_provider: npq_lead_providers.sample)
-    .build
-    .accept_application
-    .add_declaration
-end
+  # Create rejected NPQ applications
+  seed_quantity(:npq_applications_rejected).times do
+    NewSeeds::Scenarios::NPQ
+      .new(
+        lead_provider: npq_lead_providers.sample,
+        cohort:,
+      )
+      .build
+      .reject_application
+  end
 
-# Create rejected NPQ applications
-seed_quantity(:npq_applications_rejected).times do
-  NewSeeds::Scenarios::NPQ
-    .new(lead_provider: npq_lead_providers.sample)
-    .build
-    .reject_application
-end
+  # Create pending NPQ applications to ASO NPQ course
+  seed_quantity(:npq_applications_pending_aso).times do
+    NewSeeds::Scenarios::NPQ
+      .new(
+        lead_provider: npq_lead_providers.sample,
+        npq_course: NPQCourse.find_by(identifier: "npq-additional-support-offer"),
+        cohort:,
+      )
+      .build
+  end
 
-# Create pending NPQ applications to ASO NPQ course
-seed_quantity(:npq_applications_pending_aso).times do
-  NewSeeds::Scenarios::NPQ
-    .new(
-      lead_provider: npq_lead_providers.sample,
-      npq_course: NPQCourse.find_by(identifier: "npq-additional-support-offer"),
-    )
-    .build
-end
+  # Create pending NPQ applications to EHCO NPQ course
+  seed_quantity(:npq_applications_pending_ehco).times do
+    NewSeeds::Scenarios::NPQ
+      .new(
+        lead_provider: npq_lead_providers.sample,
+        npq_course: NPQCourse.find_by(identifier: "npq-early-headship-coaching-offer"),
+        cohort:,
+      )
+      .build
+  end
 
-# Create pending NPQ applications to EHCO NPQ course
-seed_quantity(:npq_applications_pending_ehco).times do
-  NewSeeds::Scenarios::NPQ
-    .new(
-      lead_provider: npq_lead_providers.sample,
-      npq_course: NPQCourse.find_by(identifier: "npq-early-headship-coaching-offer"),
-    )
-    .build
-end
+  # Create NPQLeadership applications with participant profile and declaration
+  seed_quantity(:npq_applications_eligible_for_transfer).times do
+    NewSeeds::Scenarios::NPQ
+      .new(
+        lead_provider: npq_lead_providers.sample,
+        npq_course: NPQCourse.find_by(identifier: Finance::Schedule::NPQLeadership::IDENTIFIERS.sample),
+        cohort:,
+      )
+      .build
+      .accept_application
+      .add_declaration
```

Comment:
Out of curiosity, what is the specific scenario required for having an application with a declaration and statement line item? As I was hoping that new applications that come in, we could make declarations to etc.